ml/pre-processing/get_features_from_peinfo.scala

```diff
@@ -0,0 +1,56 @@
+import com.datastax.spark.connector._
+import play.api.libs.json.Json
+import play.api.libs.json._
+import java.io.{ByteArrayOutputStream, ByteArrayInputStream}
+import java.util.zip.{GZIPOutputStream, GZIPInputStream}
+
+case class peinfo_results_by_service_name_class(service_name: String, sha256: String)
+case class peinfo_results_by_sha256_class(sha256: String, service_name: String, results: Array[Byte])
+case class peinfo_join_results_class(sha256: String, service_name: String, results: String)
+case class peinfo_int_final_seq_rdd_class(sha256: String, peinfo_int_seq_results: Array[Double])
+case class peinfo_binaray_final_array_rdd_class(sha256:String, array_results :Array[Double])
+
+def unzip(x: Array[Byte]) : String = {      
+    val inputStream = new GZIPInputStream(new ByteArrayInputStream(x))
+    val output = scala.io.Source.fromInputStream(inputStream).mkString
+    return output
+}
+
+val List17 = Array.iterate(0.0,17)(a=>a*0)
+def findAllIntinPEINFO( peinfo_json_results : JsLookupResult, time: Double): Array[Double]= {
```

Comment:
Hey, George. Sorry to reply you later.
I only extract four kinds of sections (.text / .data / .rsrc / .rdata). The other section are discarded. 
I do this because the code of Bojan's also only extracts these four kinds of sections and discards others.
What do you think about the sections with your experience?  It is not hard to add other sections into my code. 
BTW, I might have deflected your question because I dont understand this express in your words.
> multiple sections of the same type.

My understanding is .text section and .data section are multiple sections, however, I dont know the meaning of the same type.